sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java

```diff
@@ -336,28 +351,33 @@ private final synchronized void startServices(Class<? extends MobileCenterServic
             return;
         }
 
+        List<String> startedServices = new ArrayList<>();
```

Comment:
Optional as was missing before, but can you either add a block comment or remove blank line?